tests/unit/components/sl-calendar-test.js

```diff
@@ -87,30 +63,34 @@ test( 'Lock mode prevents changing state', function( assert ) {
     );
 });
 
-test( 'Clicking a day with a valid content value sends data', function( assert ) {
-    const testDay = 1;
-    const testMonth = 1;
-    const testYear = 2015;
-    const testDate = new Date( testYear, testMonth - 1, testDay );
-
-    this.subject({
-        action: 'test',
-        content: [ { date: testDate } ],
-        currentMonth: testMonth,
-        currentYear: testYear,
-
-        targetObject: {
-            test: dateContent => {
-                assert.equal(
-                    dateContent[ 0 ].date,
-                    testDate,
-                    'Date content received'
-                );
-            }
-        }
-    });
+skip( 'currentMonthString - current month string formatted as full word (January, November, ...)', function() {
+    /* https://github.com/softlayer/sl-ember-components/
```

Comment:
Please remove, as IBM's code auditing tools will flag a reference to a url in a comment.  Plus the test name should be enough direction and if it's not then it needs to be made more precise.